src/frontend/Komponenter/Behandling/Inngangsvilkår/Aleneomsorg/Bosted.tsx

```diff
@@ -38,27 +38,43 @@ const utledBostedTekst = (harDeltBosted: boolean, harSammeAdresse: boolean | und
     }
     return 'Ikke registrert på brukers adresse';
 };
-const popoverContent = (deltBosted: IDeltBostedPeriode) => (
+const historiskTekst = (historisk: boolean) => {
```

Comment:
Skjønte jeg det riktig så trekker du ut denne funksjonen til en felles funksjon slik at den slipper å bli skrevet to ganger. Husk å se på endringen jeg foreslo ovenfor 👍

Reply:
fixd